src/block_writer.rs

```diff
@@ -46,6 +46,7 @@ impl Default for BlockWriterBuilder {
 /// The key values are appended one after the other in the block, these entries are
 /// followed by an index that indicates the offset of some of the keys. This footer
 /// index stores an offset every `index_key_interval`.
+#[derive(Clone)]
```

Comment:
I think you don't use it, not sure.

Reply:
I do when I use the `vec![]` macro, the element on the left is cloned.